.coveragerc

```diff
@@ -22,7 +22,9 @@ cover_pylib = false
 # https://coverage.rtfd.io/en/latest/contexts.html#dynamic-contexts
 # dynamic_context = test_function  # conflicts with `pytest-cov` if set here
 parallel = true
-plugins = Cython.Coverage
+plugins =
+  covdefaults
```

Comment:
See https://github.com/aio-libs/yarl/pull/1084#discussion_r1741265935

Reply:
@Dreamsorcerer we don't have a `NoReturn` in this project. And I checked that the coverage change is meaningful. So I think it's fine to have this plugin enabled in at least this repo.

Reply:
But you're right that it'd exclude the entire functions, which is harmful where `NoReturn` is used. I filed an issue upstream @ https://github.com/asottile/covdefaults/issues/152 to see if this can be fixed.

Reply:
Yeah, Anthony can be like that. But there's a chance he'd accept it if somebody else did the work.

Reply:
Given the above, I'm going to integrate the plugin here and maybe other libraries, but not aiohttp for now. I may try to get some fixes upstream and re-evaluate later.

Marking the thread as resolved.